aea/cli/upgrade.py

```diff
@@ -50,7 +50,7 @@
     help_remote="For fetching packages only from remote registry.",
 )
 @click.pass_context
-@check_aea_project
+@check_aea_project(check_aea_version=False)
```

Comment:
now when running `aea upgrade`, aea version is not checked (but we do check fingerprints!)